submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -47,18 +49,19 @@ class Dog extends Animal {
 class Cat extends Animal {
    static say = 'moew';
    constructor(name, gender) {
-      super('cat', name, gender, Cat.say);
-
+      super('cat', name, gender);
+      this.say = Cat.sayingPhrase();
```

Comment:
`Cat.sayingPhrase` will return a primitive.
Look at getters/setters - they are quite interesting.... things.
I realize this task is untrivial. We can skip it. Understanding of untriviality and what doesn't really work in OOP (or at least doesn't have obvious solutions) is a great learning per se. So feel free keeping the code here as it is and dive deeper some time later.